packages/esm-appointments-app/src/patient-queue/visit-form/visit-form.component.tsx

```diff
@@ -65,7 +65,7 @@ const VisitForm: React.FC<VisitFormProps> = ({ patientUuid, appointment }) => {
   const [visitType, setVisitType] = useState<string | null>(null);
   const state = useMemo(() => ({ patientUuid }), [patientUuid]);
   const allVisitTypes = useVisitTypes();
-  const { mutate } = useAppointments(selectedDate);
```

Comment:
This is the only place that was still using the useAppointments hook, and it was only using it to get the mutate function, which seemed entirely incorrect.  Changed it to use the new mutate appointments hook we made, and then was able to get rid of the unused useAppointments.s entirely